app/analysis/index/controller.js

```diff
@@ -84,27 +84,27 @@ export const AnalysisQueryParams = new QueryParams({
     deserialize: deserializeMoment,
   },
   review: {
-    defaultValue: "",
+    defaultValue: null,
```

Comment:
Without *nulling* those QPs would also appear in the URL. Which is not intended. :(